### ISSUE TITLE
releaser composite action: doesn't work on windows runner

### ISSUE THREAD

**Issue by jeremyd2019** [pyTooling/Actions#26]
On 'windows-2022'

```
Run D:\a\_actions\pyTooling\Actions\main\releaser/composite/../releaser.py
  D:\a\_actions\pyTooling\Actions\main\releaser/composite/../releaser.py
  shell: C:\Program Files\Git\bin\bash.EXE --noprofile --norc -e -o pipefail {0}
  env:
    pythonLocation: C:\hostedtoolcache\windows\Python\3.8.10\x64
    MSYSTEM: MSYS
    INPUT_TOKEN: ***
    INPUT_FILES: staging/mingw/clangarm64/*
    INPUT_TAG: repo
    INPUT_RM: true
    INPUT_SNAPSHOTS: true
D:\a\_temp\193f3dee-d513-4616-810b-1c5c0c0b4d76.sh: line 1: D:a_actionspyToolingActionsmainreleaser/composite/../releaser.py: No such file or directory
Error: Process completed with exit code 127.
```

**Comment by umarcor** [comment]
For context, the Action was never tested on environments other than `ubuntu-latest`. On GitHub Actions, containers are only supported on Linux environments.
The composite version was written recently, in order to try working around the stability issues when uploading artifacts as assets. We wanted to evaluate whether the usage of docker had anything to do. That didn't work, so we now changed to using a pre-built container image, including GitHub's official CLI.
Therefore, the composite version is mostly untested, particularly on Windows/MSYS2.

Anyway, it'd be good to enhance it to work on non-Linux hosts!

It seems that the problem is how GitHub Actions' runner is parsing the YAML, then generating a shell script and executing it? Can you try using forward slashes instead of backslashes in "pythonLocation"?

**Comment by jeremyd2019** [comment]
pythonLocation is set by the setup-python action, not something I set myself.

However, it worked after I quoted the path to runner.py in the action.  It required 3x single quotes, the outer being YAML single quotes, and then doubled single quotes to escape them from YAML and get them through to bash.

I went for the composite version to try to avoid having to upload the files as artifacts, spawn a new ubuntu runner, download the artifacts, just to upload them again.

**Comment by umarcor** [comment]
Gotcha. It seems that docker hub is down at the moment, and that's preventing CI from running. As soon as that's fixed, I will merge your PR.

FTR, another option would be to run the Python script directly, instead of using the composite action. There is a single Python dependency (`PyGithub`), apart from the GitHub CLI.

**Comment by umarcor** [comment]
Now merged. @jeremyd2019, can you please try `pyTooling/Actions/releaser@dev`?

**Comment by jeremyd2019** [comment]
:+1:, for `pyTooling/Actions/releaser/composite@dev`: https://github.com/msys2-arm/staging/actions/runs/1604482095/workflow

**Comment by umarcor** [comment]
Thanks! Now released `v0.2.3` and bumped `pyTooling/Actions/releaser/composite@r0`.
